thali/thalireplicationmanager.js

```diff
@@ -192,11 +206,44 @@ function muxServerBridge(tcpEndpointServerPort) {
   var serverPlex = multiplex({}, function(stream, id) {
     var clientSocket = net.createConnection({port: tcpEndpointServerPort});
     stream.pipe(clientSocket).pipe(stream);
+
+    stream.on('error', function (err) {
+      console.log('Multiplex stream error %s', err);
```

Comment:
I'm confused. Now we don't seem to be tracking any errors what so ever on stream. That doesn't sound right.